provider-utils/awscloudformation/service-walkthroughs/vod-push.js

```diff
@@ -158,7 +158,7 @@ async function serviceQuestions(context, options, defaultValuesFilename, resourc
       }
 
       // determine the outputRendition of the template (HLS or DASH)
-      const currentTemplate = JSON.parse(fs.readFileSync(`${pluginDir}/templates/${advTemplate.encodingTemplate}`, { encoding: 'utf8', flag: 'r' }));
+      const currentTemplate = jobTemplate.JobTemplate;
 
       for (let counter = 0; counter < currentTemplate.Settings.OutputGroups.length; counter++) {
         if (currentTemplate.Settings.OutputGroups[0].OutputGroupSettings.Type.includes('DASH')) {
```

Comment:
Here we have to replace 0 by counter if I'm not wrong

Reply:
```diff
for (let counter = 0; counter < currentTemplate.Settings.OutputGroups.length; counter++) { 
- if (currentTemplate.Settings.OutputGroups[0].OutputGroupSettings.Type.includes('DASH')) { 
+ if (currentTemplate.Settings.OutputGroups[counter].OutputGroupSettings.Type.includes('DASH')) { outputRendition.push('DASH'); 
- } else if (currentTemplate.Settings.OutputGroups[0].OutputGroupSettings.Type.includes('HLS')) { 
+ } else if (currentTemplate.Settings.OutputGroups[counter].OutputGroupSettings.Type.includes('HLS')) { outputRendition.push('HLS'); } 
}
```

Reply:
Good catch :)